bans-core/src/main/java/space/arim/libertybans/core/commands/CommandsCore.java

```diff
@@ -131,7 +131,7 @@ public void execute(CmdSender sender, CommandPackage command) {
 		Stream<String> completions = subCommand.suggest(sender, firstArg, argIndex);
 		String lastArg = args[args.length - 1];
 		if (!lastArg.isEmpty()) {
-			completions = completions.filter((completion) -> completion.startsWith(lastArg));
+			completions = completions.filter((completion) -> completion.toLowerCase().startsWith(lastArg.toLowerCase()));
```

Comment:
```suggestion
			completions = completions.filter((completion) -> completion.toLowerCase(Locale.ROOT).startsWith(lastArg.toLowerCase(Locale.ROOT)));
```
I've two suggestions here:
1. Case comparisons should be locale-independent for consistent behavior.
2. Take `lastArg.toLowerCase(Locale.ROOT)` outside of the lambda so that it is not called multiple times.